code/game/objects/items/food/pastries.dm

```diff
@@ -751,19 +752,28 @@
 			desc = "A fat stack of fluffy pancakes!"
 		if(7 to 9)
 			desc = "A grand tower of fluffy, delicious pancakes!"
-		if(PANCAKE_MAX_STACK to INFINITY)
+		if(PANCAKE_MAX_STACK)
 			desc = "A massive towering spire of fluffy, delicious pancakes. It looks like it could tumble over!"
+		if(11 to 99)
+			desc = "A completely mind bogglingly tremendous amount of fluffy and wonderful pancakes! It towers over the realm of the mortals, watching the petty squabbles of day to day from it's grandiose pile of battery goodness."
+		if(PANCAKE_LORD_MAX_STACK)
+			desc = "One hundred pancakes. Your mind begins to melt at the sight of one hundred stacked pancakes. Only a master- no, a <b>GOD</b> could stack one hundred pancakes. The amount of milk, flour, and sugar in this fluffy monstrosity would make a bovine creature drop dead, and an entire field of wheat wither."
```

Comment:
```suggestion
			desc = "One hundred pancakes. Your mind begins to melt at the sight of one hundred stacked pancakes. Only a master-no, a <b>GOD</b> could stack one hundred pancakes. The amount of milk, flour, and sugar in this fluffy monstrosity would make a bovine creature drop dead, and an entire field of wheat wither."
```

Reply:
Disagree with this change, the - is cutting off a phrase, it's not a hyphenated compound word.

Reply:
yea silly change

Reply:
i updated it what about two hyphens

Reply:
I will accept two hypens in return for removing all Oxford commas and one cookie.

Reply:
no deal sir

Reply:
The oxford commas are incredibly important to emphasize the one hundred pancakes :)

Reply:
If it's a subordinate clause then it can be surrounded by two dashes: "Only a master—no, a god—could stack one hundred pancakes." Some choose to be represented this by two hyphens (--), or an em dash  (—). This is not the same thing as a hyphen. You can also use a dash to separate two independent clauses, but that isn't what this is doing.

Reply:
okay i like that idea a LOT